src/nft/components/collection/CollectionPageSkeleton.tsx

```diff
@@ -3,27 +3,37 @@ import { Column, Row } from 'nft/components/Flex'
 import { useIsMobile } from 'nft/hooks'
 import { CollectionBannerLoading } from 'nft/pages/collection'
 import { COLLECTION_BANNER_HEIGHT } from 'nft/pages/collection'
+import * as styles from 'nft/pages/collection/index.css'
```

Comment:
nit: I think it's better to use named imports instead of `* as styles` for better tooling support and potential tree shaking in the future. Easier to move things around too.